webview-ui/src/components/chat/__tests__/ErrorRow.spec.tsx

```diff
@@ -64,7 +64,14 @@ describe("ErrorRow", () => {
 
 		it("shows login button when showLoginButton is true and onLoginClick is provided", () => {
 			const onLoginClick = vi.fn()
-			render(<ErrorRow type="error" message="Test error message" showLoginButton={true} onLoginClick={onLoginClick} />)
```

Comment:
this file neems kilocodechange markers, but it seems just like unnecessary formatting changes?